pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/blocks/TransferableBlock.java

```diff
@@ -217,8 +217,7 @@ public DataBlock.Type getType() {
    *
    * @return whether this block is the end of stream.
    */
-  // TODO: Update the name to isTerminateBlock.
```

Comment:
This is a very old `TODO` (added in 2022), and since everyone is used to `isEndOfStreamBlock()` and `isSuccessfulEndOfStreamBlock()`, renaming it could cause more confusion. We can probably just remove this TODO

Reply:
cc @gortiz for opinion

Reply:
I may be biased, but I agree with Jackie. The TODO was subjective and given we are used to the current term, I would suggest to keep it that way and just remove the TODO

Reply:
Sounds good, will revert the refactor and just remove the todo